wellplated/tests.py

```diff
@@ -130,6 +130,28 @@ def test_container_dot_well_label(mocker: MockerFixture) -> None:
     ]
 
 
+def test_container_dot_fields() -> None:
+    """
+    Unset Container fields must return None.
+
+    This keeps the Django admin and Wagtail manage interfaces happy.
+    """
+    assert Container().code is None
+    assert Container().format is None
+
+
+def test_container_dot_methods() -> None:
+    """
+    Missing Container methods must raise AttributeError.
+
+    This keeps ClusterableModel happy.
+    """
+    with raises(AttributeError):
+        Container().get_source_expressions()  # type: ignore[misc,operator]
+    with raises(AttributeError):
```

Comment:
I like to use `match` kwarg to tighten up these assertions further, I have seen drift in errors with same type happen before

Also, feel free to ignore this

Reply:
https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.raises thanks for the tip